### ISSUE TITLE
soho toolbar flex button: redundant aria disabled

### ISSUE THREAD

**Issue by JBismonte** [infor-design/enterprise#6339]
**Describe the bug**
 aria-disabled is added in addition to the disabled attribute which is redundant and not necessary. 

**To Reproduce**

1. Go to ['this link'](http://localhost:4000/components/toolbar-flex/example-index.html)
2. Check the redundant aria disabled on the disabled button

**VPAT Recommendation**
Remove the redundant aria-disabled from all of the elements where it is redundant with disabled attribute

**Version**

- ids-enterprise: [e.g. v4.9.0 or v4.10.0] 4/64

**Screenshots**
![image](https://user-images.githubusercontent.com/29673378/165062759-df2d03de-e5d3-4e39-95e7-b6960c5da9b3.png)

**Platform**
 - Infor Application/Team Name: Landmark Client
 - Device: (if applicable): NA
 - OS Version: Windows 10
 - Browser Name: Firefox
 - Browser Version: [e.g. 22, 66.0.3359.181 (Official Build) (64-bit)]

**Additional context**
[Jira link.](https://jira.lawson.com/browse/LMCLIENT-40130)

**Comment by tmcconechy** [comment]
@JBismonte yep good catch. No need for an aria-disabled if it has a disabled

**Comment by tmcconechy** [comment]
Fix for this is - if a disabled attribute you do not need aria-disabled.
